lib/play_game.rb

```diff
@@ -0,0 +1,24 @@
+def play_game
+
+  erb(:game)
+
+  @player_1_choice = $player_1c
+  if @player_1_choice
+    @player_2_choice = ["rock", "paper", "scissors"].sample
+  end
+  if (@player_1_choice == "rock" && @player_2_choice == "rock" ||
```

Comment:
Could be refactored this to just ' if @player_1_choice == @player_2_choice ' instead of checking all possibilities

Reply:
definitely. Thanks for spotting this!